lib/webauthn/attestation_statement/tpm.rb

```diff
@@ -34,6 +34,7 @@ def valid?(authenticator_data, client_data_hash)
               OpenSSL::Digest.digest(cose_algorithm.hash_function, att_to_be_signed)
             ) &&
             matching_aaguid?(authenticator_data.attested_credential_data.raw_aaguid) &&
+            certificate_chain_trusted? &&
```

Comment:
FWIW, just opened issue https://github.com/cedarcode/tpm-key_attestation/issues/2 to move this to the extracted gem.

Reply:
Great! I'll focus on packed/u2f/safetynet/android keystore here (when time permits) :)

Reply:
TPM PR: https://github.com/cedarcode/tpm-key_attestation/pull/4